### ISSUE TITLE
X3 multi or x2-x3 multi

### ISSUE THREAD

**Issue by thedevient** [ArcadeHustle/X3_Multi_Eye_for_an_Eye#2]
Hiya, I’ve just finished the download and put the image onto a new ssd disk, it boots up fine but doesn’t look like the one in the video which has the nesicalive2 roms, this one seems to be setup for only nesica games  
![136FDCAF-F18F-4762-971D-079DFE18690E](https://user-images.githubusercontent.com/69433952/89948035-4e0f2880-dc1d-11ea-8b3e-48d2ab399e06.jpeg)
![E92C02E6-313E-4FA6-A930-0C9E57A214B1](https://user-images.githubusercontent.com/69433952/89948040-4ea7bf00-dc1d-11ea-8b0a-87c52ebcf39d.jpeg)

**Comment by ArcadeHustle** [comment]
Only the Fastio version has NxL2, and for now we are only sharing the JVS version. It is pretty trivial to bang out the rest. We are going to give the community a moment to step up and build on what we've shared before pumping out more.

**Comment by mantis151** [comment]
i have a taito fast IO is this why my controls arent working?

**Comment by thedevient** [comment]
I had a thought last night whilst downloading another rom, if this is a jvs version and nxl2 games are fast I/o am I right in thinking I’m not going to get these roms working on this whatever I try.

**Comment by mantis151** [comment]
> I had a thought last night whilst downloading another rom, if this is a jvs version and nxl2 games are fast I/o am I right in thinking I’m not going to get these roms working on this whatever I try.

I’m thinking the fast IO version is a different img file and will work with all the games including nesica2 games. I think we just have to be patient matey. There’s a couple forums discussing The multi one is Neo Arcadia

**Comment by thedevient** [comment]
Yeah I’ve joined the neo Arcadia forum to follow any progress, I have managed to get a few roms working on this like arcana hearts 2 which is helpful for those who don’t have an x2 but personally I do prefer my tx2 setup so will be sticking to that and leaving my tx3 for nxl2 games as and when available.

**Comment by ArcadeHustle** [comment]
@mantis151 yes... these images are JVS only. The FastIO version is currently private.

**Comment by ArcadeHustle** [comment]
@thedevient "no matter what I try", isn't exactly correct. We've handed you the keys to the kingdom. With some effort you would indeed figure it out. We just aren't handing that one out on a silver platter at this point in time.

**Comment by thedevient** [comment]
I’m still working on it and downloaded another rom, I’m no good at coding but I’ll still look at the format and see if there’s any way of adding the other roms, my main query was if it’s coded for jvs and not fast I/o and the nxl2 games require fast I/o am I likely to just keep hitting dead ends or is it possible for them to work with a bit of manipulation.

**Comment by ArcadeHustle** [comment]
JVS support is literally a DLL swap...

**Comment by ArcadeHustle** [comment]
Ya all sick of complaining about FastIO yet? https://github.com/ArcadeHustle/X3_Multi_Eye_for_an_Eye/issues/11#issuecomment-729825310

**Comment by GamingMackie** [comment]
> I’m still working on it and downloaded another rom, I’m no good at coding but I’ll still look at the format and see if there’s any way of adding the other roms, my main query was if it’s coded for jvs and not fast I/o and the nxl2 games require fast I/o am I likely to just keep hitting dead ends or is it possible for them to work with a bit of manipulation.

you can have all games working with the jvs image. JVS has a little bit more input lag than FastIO so you might prefer the FIO route down the road, but otherwise you can have everything.
